### PR TITLE
Deepwalk example: cpu multi-thread training, multi-classification

### PR DESCRIPTION
**Pull request by WeiyueSu**

### CODE REVIEW

File: examples/deepwalk/model.py [hunk 1]
```diff
@@ -42,25 +51,24 @@ def forward(self, src, dsts):
...
-        batch_size = dst_shape[0]
-        neg_num = dst_shape[1] - 1
+        neg_logits = paddle.matmul(
+            src_embed, neg_embed,
+            transpose_y=True)  # [batch_size, 1, neg_num]
 
-        pos_label = paddle.ones([batch_size, 1, 1], "float32")
-        neg_label = paddle.zeros([batch_size, 1, neg_num], "float32")
-        label = paddle.concat([pos_label, neg_label], -1)
+        ones_label = pos_logits * 0. + 1.
```
Comment (Yelrose) [inline examples/deepwalk/model.py:65]: paddle.ones_like ?

File: examples/deepwalk/model.py [hunk 2]
```diff
@@ -42,25 +51,24 @@ def forward(self, src, dsts):
...
+        ones_label.stop_gradient = True
+        pos_loss = self.loss(pos_logits, ones_label)
 
-        pos_weight = pos_label * neg_num
-        neg_weight = neg_label + 1
-        weight = paddle.concat([pos_weight, neg_weight], -1)
+        zeros_label = neg_logits * 0.
```
Comment (Yelrose) [inline examples/deepwalk/model.py:69]: zeros_like

Comment (WeiyueSu) [reply examples/deepwalk/model.py]: fixed.

File: examples/deepwalk/model.py (hunk 1 above)
Comment (WeiyueSu) [reply examples/deepwalk/model.py]: fixed.

**Review by Yelrose — APPROVED** [review]

**Pull request merged**